### ISSUE TITLE
Multi-line auto completion does not go away / get overwritten

### ISSUE THREAD

**Issue by potatoqualitee** [PowerShell/PSReadLine#2727]
## Environment

```
PS version: 5.1.19041.1023
PSReadline version: 2.1.0
os: 10.0.19041.1 (WinBuild.160101.0800)
PS file version: 10.0.19041.1 (WinBuild.160101.0800)
HostName: ConsoleHost (Windows Terminal)
BufferWidth: 118
BufferHeight: 29
```

## Exception report

None

## Steps to reproduce

Add the following to your profile and reload

`Set-PSReadlineOption -PredictionSource History`

Run a super long line of code 

```
f; Get-DbaDatabase -SqlInstance workstation | Out-null; $a = Connect-DbaInstance -SqlInstance localhost -SqlConnectionOnly; Get-DbaDatabase -SqlInstance workstation | Out-null; $b = Connect-DbaInstance -SqlInstance workstation -NonPooledConnection; $c = Connect-DbaInstance -SqlInstance workstation -NonPooledConnection; Get-DbaConnectedInstance | Disconnect-DbaInstance
```

Type the first letter of that long line

![image](https://user-images.githubusercontent.com/8278033/128495266-e60ecc61-5e4e-4d42-a454-8a03974fa548.png)

Execute just the first part

![image](https://user-images.githubusercontent.com/8278033/128495314-12b1f7aa-ed67-4184-9843-ff26920f8e8e.png)

## Expected behavior

The suggestion completely disappears

## Actual behavior

The suggestion hangs out but does not execute or anything, it just garbles the terminal a bit

**Comment by daxian-dbw** [comment]
@potatoqualitee Thanks for reporting this. It has been fixed in 2.2-beta versions. Please use the latest 2.2.0-beta3 version of PSReadLine.

**Comment by potatoqualitee** [comment]
awesome, thank you, can confirm that version has no issue!
